pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java

```diff
@@ -137,6 +137,9 @@ public void testHardcodedQueries()
     query = "SELECT MAX(ArrTime), MIN(ArrTime) FROM mytable WHERE DaysSinceEpoch >= 16312";
     testQuery(query, Arrays.asList("SELECT MAX(ArrTime) FROM mytable WHERE DaysSinceEpoch >= 15312",
         "SELECT MIN(ArrTime) FROM mytable WHERE DaysSinceEpoch >= 15312"));
+    query =
+        "SELECT SUM(TotalAddGTime) FROM mytable WHERE DivArrDelay NOT IN (67, 260) AND Carrier IN ('F9', 'B6') OR DepTime BETWEEN 2144 AND 1926";
```

Comment:
Is it the test case mentioned in the description section? It doesn't seem matched with `SELECT ... WHERE (filterA OR filterB) AND filterC` though.

Reply:
@jackjlli Yes, for hybrid use case, there will be an implicit time filter appended. This is the query that failed before.
Do we have an issue opened for this?